packages/cli-kit/src/node/cli.ts

```diff
@@ -52,4 +54,50 @@ export async function runCreateCLI(options: RunCLIOptions) {
   await runCLI(options)
 }
 
+export async function replaceGlobalCLIWithLocal(filepath: string): Promise<boolean> {
```

Comment:
Nitpick: What about renaming this one to: `runLocalShopifyIfPresent`

Reply:
Agreed the method name needed improvement. I went with `useLocalCLIIfDetected`